configure

```diff
@@ -1499,6 +1506,9 @@ check_compiler()
 				#blacklistcc_add "zen"
 				: # explicit no-op since bash can't handle empty loop bodies.
 			fi
+			if [ ${cc_major} -lt 6 ]; then
```

Comment:
Clang 6 is Apple clang 10. You'll need to add that condition to Line 1486.

Reply:
I am not willing to blacklist a compiler I cannot test.  When I have an Apple system with Ice Lake, I'll revisit this.  I am not going to devote any effort to dealing with Apple's breaking of the LLVM toolchain versioning.